### PR TITLE
Feat/avatar

### PR DESCRIPTION
**Pull request by ajay-mandaviya**
feat : add avatar component

### CODE REVIEW

**Comment by ajay-mandaviya** [comment]
Screenshot from 2022-02-05 13-43-50

File: component/avatar/avatar.html [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="../../style.css" />
+    <link rel="stylesheet" href="avatar.css" />
+    <title>Avatar</title>
+  </head>
+  <body>
+    <nav class="flex_space nav">
```
Comment (junaidshaikh-js) [inline component/avatar/avatar.html:12]: `nav` can be nested inside `header` element.

**Review by junaidshaikh-js — COMMENTED** [review]
Good Work!

**Review by tanishk15sharma — APPROVED** [review]
looks good to me

**Pull request merged**